examples/samplers/DEMetropolisZ_EfficiencyComparison.ipynb

```diff
@@ -4,15 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>Why is the ESS percentage &gt;100 here?</p><p> </p><p>IIRC this can happen because NUTS draws "anticorrelated" in comparison to random sampling, but maybe still double check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>the numbers square with the ESS calculation.  I added a link to the discussion on this topic on Discourse (which itself links to a Stan discussion).  </p>